README.md

```diff
@@ -21,12 +21,6 @@ Repository Content
 
 This GitHub repo is used for several things:
 
-- A backport of the `typing` module for older Python versions (2.7 and
```

Comment:
We should have a note acknowledging that the typing backport used to exist in this repo, with a mention of the last released version and a link to PyPI.

Reply:
Done.